scripts/packages/VSCodeServer/src/VSCodeServer.jl

```diff
@@ -48,12 +48,10 @@ include("../../../error_handler.jl")
 include("misc.jl")
 include("trees.jl")
 include("repl.jl")
-include("gridviewer.jl")
-include("repl_protocol.jl")
```

Comment:
I think we should keep the `repl_protocol.jl` file and its content as is. That pattern is used throughout the VS Code and our code base, i.e. that we have one file that defines the interface/protocol structs and message types, so that one can find them easily in one place.

So maybe make this PR such that it only moves the actual handler functions into individual files?

Reply:
huh, sounds fair. I personally prefer to keep protocol/handler to live closer to each other, but the consistency with other parts in our code base would be more important ;)
Will revert back the change.